src/schema/schema_manager.py

```diff
@@ -507,6 +516,25 @@ def remove_transient_and_none_values(merged_dict, normalized_entity_type):
     return filtered_dict
 
 
+"""
+Update entity via HTTP call to entity-api
+This is useful when a change in one entity necessitates changes in another, while allowing the normal triggers
+and validators to execute
+
+Parameters
+----------
+uuid : string
+    The uuid of the entity being updated
+data : dict
+    A dict representation of the json_data_dict for the updated entity
+token : string
+    The globus groups token
+"""
+def update_entity(uuid, data, token):
```

Comment:
Instead of defining this `update_status()` in schema_manager, we should handle the business logic in the trigger method. Because the schema_manager should not care about updating the entity specifically. 

We should add a getter method similar to https://github.com/hubmapconsortium/entity-api/blob/795b63d92eeee114aac28583c3bbae882e41adb2/src/schema/schema_manager.py#L1690-L1701 
so other modules can get a handle of the entity-api url and then make request inside the trigger method like this https://github.com/hubmapconsortium/entity-api/blob/795b63d92eeee114aac28583c3bbae882e41adb2/src/schema/schema_triggers.py#L1390